cypress/e2e/marc-authority/marc-authority-import.cy.js

```diff
@@ -0,0 +1,48 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import Features from '../../support/dictionary/features';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import Users from '../../support/fragments/users/users';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../support/fragments/data_import/logs/logs';
+
+describe('MARC Authority management', () => {
+  const testData = {};
+  const pathToFile = 'corporate_name(prefix_in_010Sa)sc_02.mrc';
+  const fileName = `testMarcFile.${getRandomPostfix()}.mrc`;
+  const jobProfileToRun = 'Default - Create SRS MARC Authority';
+
+  before('Creating data', () => {
+    cy.createTempUser([
+      Permissions.moduleDataImportEnabled.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+
+      cy.login(createdUserProperties.username, createdUserProperties.password, { path: TopMenu.dataImportPath, waiter: DataImport.waitLoading });
+    });
+  });
+
+  after('Deleting data', () => {
+    Users.deleteViaApi(testData.userProperties.userId);
+
+    cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading });
+    DataImport.selectLog();
+    DataImport.openDeleteImportLogsModal();
+    DataImport.confirmDeleteImportLogs();
```

Comment:
I have not found the deletion of three marc authorities that are created after the execution of the autotest.
I am talking about mark authorities with the names:
1)Apple & Honey Productions
2)Association pour la promotion et la protection de la liberté d'expression au Burundi
3)Apple Academic Press
after the execution of autotest you can found them by  this path: /marc-authorities/authorities/
you can delete it 
to remote one record you can send the api request with method DELETE to the path: records-editor/records/cde37939-fd36-47c8-a5e0-17d873b2c0d7
For example you can found the recordId in the response of the GET api request to the path: 
/search/authorities?limit=100&offset=0&query=%28keyword%3D%3D%22Apple%20Academic%22%29